examples/convert_to_aster.py

```diff
@@ -0,0 +1,104 @@
+#!/usr/bin/python
```

Comment:
This is not even a generic file for it to be named ```convert_to_aster.py```... look at the code you've provided. It is intrinsically tied to a very specific spectra... how can this be generic?